src/scripts.js

```diff
@@ -1,168 +1,167 @@
+    domUser.removeFromFavorites(specificRecipe)
   }
-}
+};
 
-function cardButtonConditionals(event) {
+const addToToCook = (event) => {
+  let specificRecipe = cookbook.recipes.find(recipe => {
+    if (recipe.id  === Number(event.target.id)) {
+      return recipe;
+    }
+  });
+  if (!event.target.classList.contains('add-active')) {
+    event.target.classList.add('add-active'); //add-active isn't a class styling
+    domUser.addToRecipesToCook(specificRecipe);
+  } else if (event.target.classList.contains('add-active')) {
+    event.target.classList.remove('add-active');
+    domUser.haveCookedThisRecipe(specificRecipe) //may want two functions here one to remove and one to cook that meal
+  };
+};
+
+const viewFavorites = () => {
+  if (cardArea.classList.contains('all')) {
+    cardArea.classList.remove('all')
+  }
+  if (!domUser.favoriteRecipes.length) {
+    showFavoritesButton.innerHTML = 'You have no favorites!';
+    populateCards(cookbook.recipes);
+    return
+  } else {
+    showFavoritesButton.innerHTML = 'Refresh Favorites'
+    cardArea.innerHTML = '';
+    domUser.favoriteRecipes.forEach(recipe => {
+      cardArea.insertAdjacentHTML('afterbegin', constructCard(recipe, "favorite-active"))
+    })
+  }
+};
+
+const cardButtonConditionals = (event) => {
   if (event.target.classList.contains('favorite')) {
     favoriteCard(event);
+  } else if (event.target.classList.contains('add-button')) {
+    addToToCook(event);
   } else if (event.target.classList.contains('card-picture')) {
     displayDirections(event);
   } else if (event.target.classList.contains('home')) {
-    favButton.innerHTML = 'View Favorites';
+    showFavoritesButton.innerHTML = 'View Favorites';
     populateCards(cookbook.recipes);
   }
 }
 
-
-function displayDirections(event) {
+const displayDirections = (event) => {
   let newRecipeInfo = cookbook.recipes.find(recipe => {
     if (recipe.id === Number(event.target.id)) {
       return recipe;
-    }
-  })
-  let recipeObject = new Recipe(newRecipeInfo, ingredientsData);
-  let cost = recipeObject.calculateCost()
-  let costInDollars = (cost / 100).toFixed(2)
-  cardArea.classList.add('all');
-  cardArea.innerHTML = `<h3>${recipeObject.name}</h3>
-  <p class='all-recipe-info'>
-  <strong>It will cost: </strong><span class='cost recipe-info'>
-  $${costInDollars}</span><br><br>
-  <strong>You will need: </strong><span class='ingredients recipe-info'></span>
-  <strong>Instructions: </strong><ol><span class='instructions recipe-info'>
-  </span></ol>
-  </p>`;
-  let ingredientsSpan = document.querySelector('.ingredients');
-  let instructionsSpan = document.querySelector('.instructions');
+    };
+  });
+  let recipeObject = new Recipe(newRecipeInfo, ingredientsData);// could we be instantiating recipes earlier in the class?
+  let cost = recipeObject.calculateCost();
+  let costInDollars = (cost / 100).toFixed(2);// this functionality could be moved into calcualte cost
```

Comment:
Yes, I think this is already in the calcluateCost method on the recipe class.

Reply:
Thank you for following up on this!